src/rtextures.c

```diff
@@ -275,6 +282,88 @@ Image LoadImageRaw(const char *fileName, int width, int height, int format, int
     return image;
 }
 
+// Load image from SVG file data with default size from SVG
+Image LoadImageSvg(const char* fileName)
+{
+    Image image = { 0 };
+
+    unsigned int dataSize = 0;
+    unsigned char* fileData = LoadFileData(fileName, &dataSize);
+
+    if (fileData != NULL)
+    {
+    	struct NSVGimage* svgImage = nsvgParse(fileData, "px", 96.0f);
+
+        const int width = (int)svgImage->width;
+        const int height = (int)svgImage->height;
+        // Delete
+        nsvgDelete(svgImage);
+
+        return LoadImageSvgWithSize(fileName, width, height);
+    }
+
+
+    return image;
+}
+
+// Load an image from SVG file data with a custom size
+Image LoadImageSvgWithSize(const char* fileName, int width, int height)
+{
+    Image image = { 0 };
+
+    unsigned int dataSize = 0;
+    unsigned char* fileData = LoadFileData(fileName, &dataSize);
+
+    if (fileData != NULL)
+    {
+        struct NSVGimage* svgImage = nsvgParse(fileData, "px", 96.0f);
+
+        
+
+        // Allocate memory for image
+        unsigned char* img = malloc(width * height * 4);
+
+        // Calculate scales for both the width and the height
+        const float scaleWidth = width / svgImage->width;
+        const float scaleHeight = height / svgImage->height;
+
+        // Set the largest of the 2 scales to be the scale to use
+    	const float scale = (scaleHeight > scaleWidth) ? scaleWidth : scaleHeight;
+
+        int offsetX = 0;
+        int offsetY = 0;
+
+        if (scaleHeight > scaleWidth)
+        {
+            offsetY = (height - svgImage->height * scale) / 2;
```

Comment:
Please, review raylib coding conventions: https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md

No spaces between `*` and `/`

Reply:
Reviewed. And I'm wondering why there are no spaces around `*` and `/` but there are spaces around `+` and `-`.

Reply:
It's a coding convention I choose for raylib, probably just a personal preference, it seems easier for my to see distinction between additions and multiplications.

Reply:
Yes, for me it emphasizes the relative precedence of the operations as well.